youngseo/.env.sample

```
@@ -0,0 +1,10 @@
+DATABASE_URL = "db_type://username:password@127.0.0.1:db_port/database_name"
+PORT = YOUR PORT NUMBER SHOULD BE HERE!
```

```suggestion
DATABASE_URL = "db_type://username:password@127.0.0.1:db_port/database_name"

PORT = YOUR PORT NUMBER SHOULD BE HERE!
```
가독성을 위해 연관이 있는 것들 끼리만 붙여주시고 관련이 없는 것들은 한칸 띄워서 작성해주세요 :)